config/initializers/config.rb

```diff
@@ -0,0 +1,12 @@
+
+config = File.join(Rails.root, 'config', 'config.yml')
+local = File.join(Rails.root, 'config', 'config-local.yml')
+
+APP_CONFIG =
+if File.exist?(local)
+  YAML.load_file(local)['settings']
+elsif File.exist?(config)
```

Comment:
I would change that to:
```ruby

APP_CONFIG = YAML.load_file(config)['settings'] || {}

if File.exist?(local)
  local_config = YAML.load_file(local)['settings'] || {}

  APP_CONFIG = APP_CONFIG.merge(local_config) do |_, global, local|
    if global.is_a?(Hash) && local.is_a?(Hash)
      global.merge(local)
    else
      local
    end
  end
end
```
In that way the local settings are merged with the default ones, overwriting them where needed.